.github/workflows/test.yml

```diff
@@ -310,9 +315,13 @@ jobs:
          - {"ipfamily": {"ip": ipv6}, "target": {"shard": {"multicast-enable": "true"}}}
          # No need to run disable-snat-multiple-gws with local GW mode
          - {"disable-snat-multiple-gws": {"enabled": "true"}, "gateway-mode": local}
+         - {"disable-snat-multiple-gws": true, "gateway-mode": local}
```

Comment:
{"disable-snat-multiple-gws": {"enabled": "true"}, "gateway-mode": local}

Reply:
Removing the line, it's a bad version of the line above